iroh-util/src/lock.rs

```diff
@@ -1,16 +1,13 @@
-use crate::exitcodes;
-use anyhow::{anyhow, Result as AnyhowResult};
-use std::fs::File;
-use std::io::prelude::*;
-use std::io::ErrorKind;
-use std::io::Write;
-use std::path::PathBuf;
-use std::process;
+use std::{fs::File, io, io::ErrorKind, io::Read, io::Write, path::PathBuf, process, result};
```

Comment:
out of curiosity, do we explicitly follow the `module` import granulariy style (as can be configured in rust-analzer)?

Reply:
no explicit style at the moment. I've been winging this, and would love to coalesce around a style choice